### ISSUE TITLE
Allow custom GitHub repo for issues

### ISSUE THREAD

**Issue by xfq** [w3c/respec#3433]
**Is your feature request related to a problem? Please describe.**

`class="issue" data-number="1"` only works for the repo in the configuration option `github`.

**Describe the solution you'd like**

It would be great if ReSpec supports embedding content of a GitHub issue in another GitHub repo.

**Describe alternatives you've considered**

Fill in the content of the issue directly and manually.

**Comment by marcoscaceres** [comment]
In theory, it should be trivial to add this... as I'm pretty sure we used to support this functionality in the past.

**Comment by sidvishnoi** [comment]
A `data-github="owner/repo"` as a companion for `data-number=issueNumber` might be a good?

**Comment by marcoscaceres** [comment]
yeah, something like that would work... then it would be on a per-issue basis.

**Comment by sidvishnoi** [comment]
I'm thinking of soft-deprecating (no warning) `.issue[data-number="issueNumber"]` and instead use `data-issue="org/repo#issueNumber"` for this feature.

`data-issue="issueNumber"` would then be equivalent to `.issue[data-number="issueNumber"]`. Authors won't need to add a `.issue` and we won't need to support  `data-github`.
`data-number` and `data-github` don't make sense without a `.issue`, so why not add a `data-issue` to merge them? WDYT @marcoscaceres?

**Comment by marcoscaceres** [comment]
I like it... but there might be quite a few specs using `<div class="issue" data-number="xxx">`... maybe we can keep backwards compat? We can do a number check?

**Comment by sidvishnoi** [comment]
Yeah. It'll be backward compat, and discouraged in docs only. No warning.
